lib/cretonne/src/verifier.rs

```diff
@@ -614,6 +615,28 @@ impl<'a> Verifier<'a> {
         Ok(())
     }
 
+    fn cfg_integrity(&self, cfg: &ControlFlowGraph) -> Result<()> {
+        for ebb in self.cfg.ebbs() {
+            let cfg_successors = cfg.get_successors(ebb);
+            let cfg_predecessors = cfg.get_predecessors(ebb);
+            for &succ in self.cfg.get_successors(ebb) {
+                if !cfg_successors.contains(&succ) {
```

Comment:
I went for doing two uses of `difference`, to distinguish the cases when we have excess members and when we have missing members. There's no `Ord` impl for `BasicBlock` though, so I can't currently use BTreeSet. Should I just throw a basic one on it for now? (order by ebb then by inst)